test/integration/globals/app-nav.js

```diff
@@ -16,15 +16,10 @@ context('App Nav', function() {
       .visit();
 
     cy
-      .get('.app-nav__header')
-      .click();
-
-    cy
-      .get('.picklist')
-      .should('contain', 'Sign Out')
-      .contains('Help & Support')
-      .should('have.attr', 'href')
-      .and('contain', 'help.roundingwell.com');
```

Comment:
I don't think this is relevant for this specific test anymore. 

I believe this is testing that the nav droplist has only the `Help & Support` and `Sign Out` links in it for non-manager users. And that it doesn't include the admin tools links, which no longer reside there.